packages/core/src/transport/transport.ts

```diff
@@ -47,6 +103,14 @@ function reportBeaconError(e: unknown) {
   }
 }
 
+let hasReportedXhrError = false
+function reportXhrIssue(xhrTransportContext: TransportIssueContext) {
+  if (!hasReportedXhrError) {
+    hasReportedXhrError = true
+    addMonitoringMessage('XHR fallback failed', xhrTransportContext)
```

Comment:
was there any thoughts on log level?
if not, I'd be in favor of being consistent with beacon and use `error` instead of `info`.

Reply:
Was thinking there is no error to capture with a stack trace (name, message, stack). It's more of a warning to let the team know network requests are dropping. No actions are necessarily involved.

Reply:
ok, got it.